client/common/cmdline.c

```diff
@@ -3173,6 +3216,77 @@ BOOL freerdp_client_load_addins(rdpChannels* channels, rdpSettings* settings)
 		    TRUE; /* these RDP8 features require rdpdr to be registered */
 	}
 
+	if (settings->DrivesToRedirect)
+	{
+
+		/*
+		 * Drives to redirect:
+		 *
+		 * Very similar to DevicesToRedirect, but can contain a
+		 * comma-separated list of drive letters to redirect.
+		 */
+		const BOOL empty = !settings->DrivesToRedirect || (strlen(settings->DrivesToRedirect) == 0);
```

Comment:
We already checked that DrivesToRedirect is not NULL. We can just do `if (strlen(settings->DrivesToRedirect) > 0)` here

Reply:
done